### ISSUE TITLE
120% score

### ISSUE THREAD

**Issue by Niek** [abrahamjuliot/creepjs#128]
The [latest update](https://github.com/abrahamjuliot/creepjs/commit/3dd8670645fcf7c2641dfb25794e6c62c9bf8faa) seem to have broken the score calculation, on a non-spoofed browser it shows as 120% now.

**Comment by abrahamjuliot** [comment]
120% was initially intended, but it does not show +20% is a reward for the low switch count. I applied a fix to make the score 100% max. The reward is still applied against any reductions from trash or lies. If the loose fingerprint switches 2 or more the reward is lost.

**Comment by Thorin-Oakenpants** [comment]
True story. I once got 101% in a history exam in high school. The exam was in essay form. The marker had already given someone else 100% when he got to mine, but mine was a class above that, so he gave me 101%.

On another note: I once got 99% in Latin (and first in class for once: there was this one guy who always beat me by 1%), and my mom's first reaction was .. why didn't you get 100%? 

And on yet another note: I got numerous 100% scores in industry standard papers, and my boss, who was paying for it, would get the results back, and then let me know. The first time I got 100%, I said "damn, I didn't get 101%) and he got upset, saying that no-one should ever be allowed 100%

so .. now I'm just like `¯\_(ツ)_/¯` ... :beers: might as well

/soz (not) for sharing 😀
